ddtrace/tracer.py

```diff
@@ -234,7 +234,7 @@ async def web_handler(request):
         return ctx
 
     def current_trace_context(self, *args, **kwargs):
-        # type (...) -> Optional[Context]
+        # type: (...) -> Optional[Context]
         """Return the context for the current trace.
 
         If there is no active trace then None is returned.
```

Comment:
I cannot add the suggestion on the line, but we need to change L242

```diff
-        active = self.context_provider.active(*args, **kwargs)
+        active = self.context_provider.active()
```

Reply:
resolved in 7263d36c011583afa7347339d4078752ee0aaf27